### ISSUE TITLE
Where is PHP documentation?

### ISSUE THREAD

**Issue by zagarskas** [plaid/idv-quickstart#13]
Is there no PHP SDK available?
I see some great languages here... Python, NodeJS, Ruby, JAVA... **but PHP didn't make that list?**

I guess I could just try to interface with the API directly using PHP CURL functions

Seems even tiny-quickstart->vanilla_js requires NODE backend?
https://github.com/plaid/tiny-quickstart/tree/main/vanilla_js

begs the questions:

1. Why no PHP backend?
2. Any docs on making calls to the API directly using PHP?
3. What options do we have on the cPanel Linux PHP stack?

Seems to me our only option is just to `curl -X POST https://sandbox.plaid.com/link/token/create \` and write our own code. (insert sad face here)

I found this: https://github.com/TomorrowIdeas/plaid-sdk-php
Which seems promising...

Basically, we need to do 1 simple thing:
 **KYC (no financial stuff)** 
we just want to verify a boolean, True/False, is the user real or not.

**Comment by ToddKerpelman** [comment]
Hi, @zagarskas . You are correct that Plaid does not have an official PHP SDK, but the @TomorrowIdeas SDK is a community-supported library for PHP that you can check out.

Note that community libraries are not officially supported by Plaid, so Plaid can't provide assistance with using these libraries or guarantee that they will be kept up-to-date. Your best bet is to reach out to the community on their github page and ask there.

(I was also going to point you to [this issue here](https://github.com/TomorrowIdeas/plaid-sdk-php/issues/86), but I see you found it already.)
